README.md

```diff
@@ -1,18 +1,26 @@
 # RealtimeSigns
 
-[![Build Status](https://semaphoreci.com/api/v1/projects/39cb0e53-0299-441e-ab09-ddcb9fa9d2aa/1806488/badge.svg)](https://semaphoreci.com/mbta/realtime_signs)
-
 Here's a general overview of the realtime_signs application which should be helpful for getting oriented to how things work. Please keep this up to date -- if you change something major in the code (build process, renaming modules, significant functionality changes, new configuration, etc.) document it here!
 
-## Building and running the app
+## Development
 
+* Run `asdf install` from the repository root.
 * `mix deps.get` to fetch dependencies.
 * At this point you should be able to run `mix test` and get a clean build.
 * To start the server, run `mix run --no-halt`. See "relevant environment variables" below for parameters you might want to pass to the server.
 
-## Relevant environment variables
+### Running locally in a docker container
+If you need to run realtime signs in a local docker container, there are a few extra steps you'll need to take.
+
+1. Follow the instructions in [this notion doc](https://www.notion.so/mbta-downtown-crossing/Creating-debugging-a-Windows-Docker-container-2f21af809c894aab8038d12ae9c54361) for your OS to set up docker
+2. Once docker is running on either your Windows machine or your Windows VM, run `docker build .` in the root directory of realtime_signs.
+3. In order to run the image, you'll need a number of env variables. It's probably easiest to add a env.list file to your local repo containing. You can read about setting env variables in a container [here](https://docs.docker.com/engine/reference/commandline/run/#set-environment-variables--e---env---env-file)
+4. Run the image in a container with `docker run --env-file env.list [IMAGE]`
```

Comment:
When I was doing the `docker run` step for the screen_checker work, it looked like `docker build .` created a nameless image and then I wasn't sure what image id to use. I had to make sure to name the image in the build step. Is that what you have to do here as well?

Reply:
Oh, I didn't even know you could name an image. I just use the last image tag provided in the output of `docker build .`